backends/s3/src/main/scala/geotrellis/s3/package.scala

```diff
@@ -1,11 +1,13 @@
 package geotrellis.spark.io
 
+import geotrellis.spark.render.RenderedImages
 import org.apache.spark.rdd.RDD
 
+
 package object s3 {
   private[s3]
   def makePath(chunks: String*) =
     chunks.filter(_.nonEmpty).mkString("/")
 
-  implicit class S3RDDExtensions[K,V](rdd: RDD[(K,V)]) extends SaveToS3Methods[K, V](rdd)
+  implicit class S3RDDExtensions[K](rdd: RenderedImages[K]) extends SaveToS3Methods[K](rdd)
```

Comment:
we're settling on `withSaveToS3Methods` type of naming pattern for these implicit classes.